build-logic/convention/src/main/kotlin/GravatarOpenApiGeneratorConventionPlugin.kt

```diff
@@ -0,0 +1,77 @@
+import org.gradle.api.Plugin
+import org.gradle.api.Project
+import org.gradle.kotlin.dsl.apply
+import org.gradle.kotlin.dsl.configure
+import org.gradle.kotlin.dsl.withType
+import org.openapitools.generator.gradle.plugin.extensions.OpenApiGeneratorGenerateExtension
+import org.openapitools.generator.gradle.plugin.tasks.GenerateTask
+
+class GravatarOpenApiGeneratorConventionPlugin : Plugin<Project> {
+    override fun apply(target: Project) {
+        with(target) {
+            apply(plugin = "org.openapi.generator")
```

Comment:
Minor, but is it possible to reference this plugin by type instead of name?

Reply:
Yes, TIL

Reply:
I see other place `GravatarAndroidLibraryConventionPlugin` that could have this suggestion applied as well

Reply:
Right! All are applied by type now.